wordpress-mobile/WordPress-Android
# [TEST] Blogging Reminders: Test scheduling logic and notification

This PR is for testing purposes only and should not be merged. It adds a dummy UI so we can more easily test the scheduling logic and notification introduced by #14719. 

To test:

1. Go to the My Site screen.
1. Scroll down and tap the "Blogging Reminders" item. 
1. Select a reminder type and days – if applicable – and then tap the FAB. 
1. Notice a new `WorkInfo` item appears in the list.

If you actually want to see the notification right away, you might need to make some manual changes to the initial delay in the `ReminderScheduler` class.
